assets/javascripts/discourse/initializers/decrypt-posts.js

```diff
@@ -212,14 +216,29 @@ function postProcessPost(siteSettings, topicId, post) {
   }
 
   // Paint category and tag hashtags
-  const unseenTagHashtags = linkSeenHashtags(post);
-  if (unseenTagHashtags.length > 0) {
-    if (!hashtagsQueue) {
-      hashtagsQueue = new DebouncedQueue(500, fetchUnseenHashtags);
+  if (!siteSettings.enable_experimental_hashtag_autocomplete) {
```

Comment:
super minor but can we revert it have 
```
if siteSettings.enable_experimental_hashtag_autocomplete
else
end
```

instead of
```
if !siteSettings.enable_experimental_hashtag_autocomplete
else
end
```

This is one less flip for brain :)

Reply:
Ah yep sounds good! :+1: